internal/db/update/schema.go

```diff
@@ -126,25 +128,30 @@ func (s *SchemaUpdate) Ensure(db *sql.DB) (int, error) {
 func (s *SchemaUpdate) Dump(db *sql.DB) (string, error) {
 	var statements []string
 	err := query.Transaction(context.TODO(), db, func(ctx context.Context, tx *sql.Tx) error {
-		versions, err := query.SelectIntegers(ctx, tx, "SELECT version FROM schemas ORDER BY version")
-		if err != nil {
-			return err
-		}
+		current := 0
+		for _, versionType := range []int{0, 1} {
```

Comment:
0 and 1 should be defined as constants imho to avoid the use of "magic numbers" in this important part of the code base.